www/pages/docs/front-matter.md

```diff
@@ -38,7 +39,7 @@ imports:
 
 ```
 
-See our [Markdown Docs](/docs/markdown#imports) for more information about rendering custom elements in markdown files.
+See our [Markdown Docs](/docs/markdown#imports) for more information about rendering custom elements in markdown files. -->
 
 
 ### Template
```

Comment:
I think it might be worth linking to the front-matter docs #page-template as well. This way both link to each as they both involve each other.